app/jobs/bulk_action_jobs.rb

```diff
@@ -272,4 +272,12 @@ def perform(collection_id, overwrite = true, save_field = nil)
       [successes, errors]
     end
   end
+
+  class ReturnCheckouts < ActiveJob::Base
+    queue_as :bulk_return_checkouts
+    def perform(collection_id, _params)
```

Comment:
I don't think the `_params` parameter is needed here.  Other jobs in BulkActionJobs follow the pattern `perform(documents, params)` since they are getting queued from the BookmarksController and having them all have the same method signature made things simpler.  But this job is only ever going to be called on one collection and won't be used in the BookmarksController so I think removing the unused param makes sense.
```suggestion
    def perform(collection_id)
```